### ISSUE TITLE
[Adaptive Cards- Web Render Test on Teams]: While navigating on the Progress bar present on the 'Poll' card, voice over is conveying irrelevant information.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#408]
### Target Platforms

iOS

### SDK Version

7.13

### Application Name

Microsoft Teams

### Problem Description

**Test Environment:**
Device: iPhone 12
iOS Version: 18.5
Teams Application Version: 7.13
Screen Reader: Voice Over

**Pre-Requisite:**
Turn on Voice Over.

**Repro Steps:**

1. Open the application using double tap.
2. Perform right swipe, navigate to the AC Web render group, and double tap to enter in the group.
3. Navigate to any poll using swipe gesture.
4. Now perform right swipe and navigate to the progress bar and observe the issue.

**Actual Result:**
Screen Reader is announcing irrelevant information while navigating on the progress bar present on the poll card. Here it is announcing the link information along with image.

**Expected Result:**
Screen Reader should announce the correct and relevant information while navigating on the progress bar. It should announce something like **"Progress indicator 22 %"**

**Note:**
Same issue is repro throughout the polls.

**User Impact:**
It will impact the screen reader user as they will not be able to know the completion of the poll if the progress indicator information is not announced by the screen reader.

**WCAG Reference:**
[Understanding Success Criterion 1.3.1: Info and Relationships | WAI | W3C](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/e0227cef-82ce-4489-b111-b0df06bcdbd5

### Card JSON

```JSON
NA
```
